PayPalCheckout/src/main/java/com/paypal/android/checkout/shipping/ShippingChangeActions.kt

```diff
@@ -0,0 +1,76 @@
+package com.paypal.android.checkout.shipping
+
+import com.paypal.android.checkout.patch.PatchOrderRequest
+
+/**
+ * The actions that can be performed when the [OnShippingChange.onShippingChanged] callback is
+ * invoked.
+ */
+class ShippingChangeActions internal constructor(private val mChangeActions: com.paypal.checkout.shipping.ShippingChangeActions) {
```

Comment:
Does this need to be hungarian notation? Seems odd since we don't use that notation anywhere else.

Reply:
i wrapped most of native checkout classes so that the interface is exactly the same to them now. Im open to discussion though if we think we need to change the name on some specific classes